src/cli/cli.ts

```diff
@@ -489,124 +484,39 @@ makeCommand({
   action: redelegateStake,
 });
 
-makeCommand<BuyRecordCLIOptions>({
```

Comment:
this PR includes a refactor that moves arns purchase actions to their own file and improves/shares their respective assertions